src/library-authoring/course-import/CourseImportTaskListItem.jsx

```diff
@@ -28,29 +28,48 @@ const CourseImportTaskListItem = ({ intl, task }) => {
   }
 
   return (
-    <div className="library-link">
-      <Row className="h-100">
-        <Col xs={12} className="my-auto">
-          <h3 className="library-title">Import of {task.course_id}</h3>
-        </Col>
-      </Row>
-      <div className="library-metadata">
-        <span className="library-state metadata-item">
-          <span className="label">{intl.formatMessage(messages['library.library.course_import.list_item.state'])}</span>
-          <span className="value">
+    <Card className="mt-1 mb-3">
+      <Card.Header
+        className="library-authoring-course-import-block-card-header"
+        title={`Import of ${task.course_id}`}
+        subtitle={
+          <>
             <Badge variant={badgeVariant}>{task.state}</Badge>
-          </span>
-        </span>
-        <span className="library-org metadata-item">
-          <span className="label">{intl.formatMessage(messages['library.course_import.list_item.organization'])}</span>
-          <span className="value">{task.org}</span>
-        </span>
-        <span className="library-slug metadata-item">
-          <span className="label">{intl.formatMessage(messages['library.course_import.list_item.created_at'])}</span>
-          <span className="value">{new Date(task.created_at).toLocaleString()}</span>
-        </span>
-      </div>
-    </div>
+            <span> • </span>
+            <span>{task.org}</span>
+            <span> • </span>
+            <span>{new Date(task.created_at).toLocaleString()}</span>
+          </>
+        }
+      />
+   <div className="library-metadata">
+
+   </div>
+    </Card>
+
+    // <div className="library-link">
+    //   <Row className="h-100">
+    //     <Col xs={12} className="my-auto">
+    //       <h3 className="library-title">Import of {task.course_id}</h3>
+    //     </Col>
+    //   </Row>
+    //   <div className="library-metadata">
+    //     <span className="library-state metadata-item">
+    //       <span className="label">{intl.formatMessage(messages['library.library.course_import.list_item.state'])}</span>
+    //       <span className="value">
+    //         <Badge variant={badgeVariant}>{task.state}</Badge>
+    //       </span>
+    //     </span>
+    //     <span className="library-org metadata-item">
+    //       <span className="label">{intl.formatMessage(messages['library.course_import.list_item.organization'])}</span>
+    //       <span className="value">{task.org}</span>
+    //     </span>
+    //     <span className="library-slug metadata-item">
+    //       <span className="label">{intl.formatMessage(messages['library.course_import.list_item.created_at'])}</span>
+    //       <span className="value">{new Date(task.created_at).toLocaleString()}</span>
+    //     </span>
+    //   </div>
+    // </div>
```

Comment:
Reminder to remove the reference material :)

Reply:
addressed in https://github.com/openedx/frontend-app-library-authoring/commit/42c82d41cd835ecf990db9a694c02cf19983f998